core/server/common/src/main/webapp/browse-body.jsp

```diff
@@ -17,35 +17,17 @@
 
 <jsp:include page="header-scripts.jsp" />
 
-<script>
-  var currentDir = "<%= request.getAttribute("currentPath").toString() %>";
-  function getInputPath() {
-    var path = $.trim($("#pathInput").val());
-    return path;
-  }
-
-  function changeDir() {
-    var path = getInputPath();
-    path = encodeURI(path);
-    window.location.href = "./browse?path=" + path;
-  }
+<link rel="stylesheet" href="css/ui-grid.css" type="text/css">
```

Comment:
@maobaolong it's very important to add the `package.json` to specify the version. I don't think it's a good practice to cache the npm module directly. Can you take a look if we can add a maven task for installing npm modules? For example, the answers in this post might be useful: http://stackoverflow.com/questions/17103748/how-to-deploy-a-node-js-app-with-maven